addon/views/multi-item-collection.js

```diff
@@ -0,0 +1,23 @@
+import Ember from 'ember';
+import StyleBindingsMixin from 'ember-table/mixins/style-bindings';
+
+export default Ember.CollectionView.extend(
+StyleBindingsMixin, {
+  styleBindings: 'width',
+  itemViewClassField: null,
+
+  createChildView: function(view, attrs) {
+    var itemViewClassField = this.get('itemViewClassField');
+    var itemViewClass = attrs.content.get(itemViewClassField);
+    if (typeof itemViewClass === 'string') {
+      if (/[A-Z]+/.exec(itemViewClass)) {
```

Comment:
This regex confuses me. Why the `+` if you're only checking if the first letter is capital?